### ISSUE TITLE
For Custom Dataset Training

### ISSUE THREAD

**Issue by saulgooodman** [RuoyuFeng/CCEdit#7]
Hello, great work! I would like to use the CCEdit method on a set of custom videos. Do I need to train every video scene separately, and how long does the training cost? Thank you!

**Comment by RuoyuFeng** [comment]
> Hello, great work! I would like to use the CCEdit method on a set of custom videos. Do I need to train every video scene separately, and how long does the training cost? Thank you!

No, you don't need to train the model on each of your videos. Directly inference would be fine.
